src/collective/polls/static/js/collective.poll.js

```diff
@@ -2,6 +2,39 @@
 
     "use strict"; // jshint ;_;
 
+    $(function() {
+        if ($('.poll-tile').length > 0) {
+            var update_tile = function($tile) {
+                $.ajax({
+                    url: location.href,
+                    context: $tile,
+                    success: function(info) {
+                        var $tile = this;
+                        var updated_info = $(info).find('[data-tile="'+$tile.attr('data-tile')+'"]');
+                        $tile.html(updated_info.html());
+                        $('.poll-data, .votePortlet form').drawpoll();
+                    }
+                });
+            };
+
+            $('.poll-tile form').on('submit', function(e) {
+                e.preventDefault();
+                var $form = $(this);
+                $.ajax({
+                   type: 'POST',
+                   url: $form.attr('action'),
+                   data: $form.serialize() + '&poll.submit=Submit',
```

Comment:
not in this case.. the [$form.serialize()](https://api.jquery.com/serialize/) returns something like:
```
'single=Single&multiple=Multiple&multiple=Multiple3&check=check2&radio=radio1'
```